lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py

```diff
@@ -27,7 +27,7 @@
 import iris.tests as tests
 
 from iris.exceptions import TranslationError
-from iris.fileformats.grib._load_convert import scanning_mode, ScanningMode
+from iris.fileformats.grib._load_convert import ScanningMode, scanning_mode
```

Comment:
I meant why had someone chosen to put '**s**canning_mode' before '**S**canningMode'. Your correction makes it follow the ascii order of things but there must have been a reason someone did the above (unless the intention was always to use ascii and these are just examples of mistakes)